### ISSUE TITLE
Do not print flow123d debug message when generating files

### ISSUE THREAD

**Issue by x3mSpeedy** [flow123d/flow123d#400]
Calling flow123d binary will **always** produce debug info message, e.g.:
```bash
$ > ./flow123d --JSON_machine 
```

will produce following output:
```json
full program name: /home/jan-hybs/Dokumenty/Smartgit-flow/build-JHy_profiler_fix/bin/flow123d
full flow prefix: /home/jan-hybs/Dokumenty/Smartgit-flow/build-JHy_profiler_fix
default py prefix: "/usr"
Python path: /home/jan-hybs/Dokumenty/Smartgit-flow/build-JHy_profiler_fix/lib/python2.7/:/home/jan-hybs/Dokumenty/Smartgit-flow/build-JHy_profiler_fix/lib/python2.7/plat-x86_64-linux-gnu:/home/jan-hybs/Dokumenty/Smartgit-flow/build-JHy_profiler_fix/lib/python2.7/lib-tk:/home/jan-hybs/Dokumenty/Smartgit-flow/build-JHy_profiler_fix/lib/python2.7/lib-old:/home/jan-hybs/Dokumenty/Smartgit-flow/build-JHy_profiler_fix/lib/python2.7/lib-dynload
Python home: /home/jan-hybs/Dokumenty/Smartgit-flow/build-JHy_profiler_fix:"/usr"
Python prefix: /home/jan-hybs/Dokumenty/Smartgit-flow/build-JHy_profiler_fix
Python exec prefix: /home/jan-hybs/Dokumenty/Smartgit-flow/build-JHy_profiler_fix
[
{
"id" : "b5e714fb8a6dbf66",
"input_type" : "Record",
"type_name" : "Root",
"type_full_name" : "Root",
"attributes" : {
},
...
```

When generating files message could be supressed.

**Comment by jbrezmorf** [comment]
Two changes should be done:
- remove uncessary python debugging info in python_loader
- write json data into a file: flow123d_ist.json

**Comment by x3mSpeedy** [comment]
Python debug info is now suppresed
JSON_machine argument now expect filename to be given, e.g.:
```bash
$ > ./flow123d --JSON_machine "flow123d_ist.json"
```

Later in build step is this used like this:
```makefile
ref-doc: cmake update-build-tree
	$(BUILD_DIR)/bin/flow123d --JSON_machine "$(DOC_DIR)/input_reference.json"
```

**Comment by jbrezmorf** [comment]
done in pull #408 
merged in 76d860fd6a
